src/android/BiometricActivity.java

```diff
@@ -0,0 +1,159 @@
+package de.niklasmerz.cordova.biometric;
+
+import android.app.Activity;
+import android.app.KeyguardManager;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+
+import com.exxbrain.android.biometric.BiometricPrompt;
+
+import java.util.concurrent.Executor;
+
+public class BiometricActivity extends AppCompatActivity {
+
+    private static final int REQUEST_CODE_CONFIRM_DEVICE_CREDENTIALS = 2;
+    private PromptInfo mPromptInfo;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setTitle(null);
+        int layout = getResources()
+                .getIdentifier("biometric_activity", "layout", getPackageName());
+        setContentView(layout);
+
+        if (savedInstanceState != null) {
+            return;
+        }
+
+        mPromptInfo = new PromptInfo.Builder(getIntent().getExtras()).build();
+        authenticate();
+
+    }
+
+    private void authenticate() {
+        final Handler handler = new Handler(Looper.getMainLooper());
+        Executor executor = handler::post;
+
+        BiometricPrompt biometricPrompt =
+                new BiometricPrompt(this, executor, mAuthenticationCallback);
+
+        BiometricPrompt.PromptInfo.Builder promptInfoBuilder = new BiometricPrompt.PromptInfo.Builder()
+                .setTitle(mPromptInfo.getTitle())
+                .setSubtitle(mPromptInfo.getSubtitle())
+                .setDescription(mPromptInfo.getDescription());
+
+        if (mPromptInfo.isDeviceCredentialAllowed()
+                && Build.VERSION.SDK_INT <= Build.VERSION_CODES.P) { // TODO: remove after fix https://issuetracker.google.com/issues/142740104
+            promptInfoBuilder.setDeviceCredentialAllowed(true);
+        } else {
+            promptInfoBuilder.setNegativeButtonText(mPromptInfo.getFallbackButtonTitle());
```

Comment:
`getFallbackButtonTitle` is only used here for the cancel button string. I am not sure if this is clear to use. On iOS this is the fallback string for "use backup" and on Android this is the "cancel" button. Should we separate this in to two parameters? If you define "Passwort benutzen" as `fallbackButtonTitle` (German for "use backup") this makes sense on iOS but no sense on Android since this is used on the cancel button.

Reply:
> On iOS this is the fallback string for "use backup" and on Android this is the "cancel" button.

I don't completely understand you. I think that the reason for using "use backup" is `disableBackup=false` but not a platform.

> Should we separate this in to two parameters?

What parameters do you want to separate.

Reply:
> > On iOS this is the fallback string for "use backup" and on Android this is the "cancel" button.
> 
> I don't completely understand you. I think that the reason for using "use backup" is `disableBackup=false` but not a platform. In another case - if `disableBackup=true` the "cancel" button is used.
> 
I am talking about this use case: I want to disable backup. So I set `disableBackup=true`. Now I want to translate the cancel button. I can only do this by setting `fallbackButtonTitle`. That is confusing to me. On iOS the cancel button is translated by the OS, isnt it?

Reply:
That's why I think we need to add another option to set the `NegativeButtonText` separately.

Reply:
@exxbrain and @greaterking Do you agree? I could do the change, too.

Reply:
@NiklasMerz Yes, please!

Reply:
I just added a paramter for the negative button. That should be good for now.

Any objections against a release?